src/utils/pageWrappers/logUncaughtErrors.js

```diff
@@ -0,0 +1,16 @@
+import logger from 'src/utils/logger'
+// A top level server side wrapper that catches all errors in the getServerSideProps func
+// and logs them through our logger
+const logUncaughtErrors = (getServerSidePropsFunc) => async (ctx) => {
+  let composedProps = { props: {} }
+  try {
+    if (getServerSidePropsFunc) {
+      composedProps = await getServerSidePropsFunc(ctx)
+    }
+  } catch (e) {
+    logger.error(e)
```

Comment:
Actually, we should throw here so Next.js shows an error page. We don't want to try to render a page with bad props.